doc/debian/jitsi-meet/jitsi-meet.conf

```diff
@@ -0,0 +1,34 @@
+# this is jitsi-meet nginx module configuration
+# this forward all http traffic to the nginx virtual host port
+# and the rest to the turn server
+#
+# Multiplexing based on ALPN is DEPRECATED. ALPN does not play well with websockets on some browsers and reverse proxies.
+# To migrate away from using it read: https://jitsi.org/multiplexing-to-bridge-ws-howto
```

Comment:
Why not link directly to the handbook?

Reply:
Changing the title for example will invalidate the anchor link. And I already used this link in the warning message when installing.

Reply:
Ack